ee/hogai/eval/ci/eval_sql.py

```diff
@@ -699,3 +699,129 @@ async def eval_sql(call_root_for_insight_generation, pytestconfig):
         data=all_cases,
         pytestconfig=pytestconfig,
     )
+
+
+@pytest.mark.django_db
+async def eval_sql_absolute_dates(call_root_for_insight_generation, pytestconfig):
+    """Test SQL generation with absolute dates, used in Deep Research reports generation."""
+
+    # Create a wrapper that calls the fixture with absolute_sql_dates=True
+    async def callable_with_absolute_dates(query_with_extra_context: str | tuple[str, str]):
+        return await call_root_for_insight_generation(query_with_extra_context, absolute_sql_dates=True)
+
+    all_cases: list[EvalCase] = [
+        ### Absolute dates
+        EvalCase(
+            input="What are the top 10 countries by number of users in the last 7 days? Use SQL",
```

Comment:
Would it add value to have cases like "last 7 days" spreading across multiple months?